LAKMobile/src/screens/DriverRegistration.tsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
```

Comment:
This doesn't seem to be matching the figma. Do we plan to merge this now and improve on it later?

Reply:
@shreekarp please resolve this comment when the driver registration stuff is removed and I can approve!